OpenRA.Game/Graphics/RgbaSpriteRenderer.cs

```diff
@@ -22,44 +22,36 @@ public RgbaSpriteRenderer(SpriteRenderer parent)
 			this.parent = parent;
 		}
 
-		public void DrawSprite(Sprite s, in float3 location, in float3 size)
+		public void DrawSprite(Sprite s, in float3 location, in float3 scale)
 		{
 			if (s.Channel != TextureChannel.RGBA)
 				throw new InvalidOperationException("DrawRGBASprite requires a RGBA sprite.");
 
-			parent.DrawSprite(s, location, 0, size);
+			parent.DrawSprite(s, 0, location, scale);
 		}
 
-		public void DrawSprite(Sprite s, in float3 location)
+		public void DrawSprite(Sprite s, in float3 location, float scale = 1f)
 		{
 			if (s.Channel != TextureChannel.RGBA)
 				throw new InvalidOperationException("DrawRGBASprite requires a RGBA sprite.");
 
-			parent.DrawSprite(s, location, 0, s.Size);
+			parent.DrawSprite(s, 0, location, scale);
 		}
 
-		public void DrawSprite(Sprite s, in float3 a, in float3 b, in float3 c, in float3 d)
+		public void DrawSprite(Sprite s, in float3 location, in float scale, in float3 tint, float alpha)
```

Comment:
Is there a reason the alpha isn't an in parameter?
Same question for the scale on the above method.

Reply:
alpha and scale are floats, which are too small for the in modifier: https://docs.microsoft.com/en-us/dotnet/csharp/write-safe-efficient-code#use-in-parameters-for-large-structs

Reply:
But `scale` has an in modifier here? 🤔

Reply:
oh, thats a typo. Will fix.